tensorflow/core/kernels/BUILD

```diff
@@ -4079,6 +4087,7 @@ filegroup(
         "fake_quant_ops.cc",
         "fifo_queue.cc",
         "fused_batch_norm_op.cc",
+        "whole_file_read_ops.cc",
```

Comment:
Thanks for the review. 

Do you mean the whole android part or the "whole_file_read_ops.cc"?
For the whole Android part, yes, I'll make it another PR.
For "whole_file_read_ops.cc", label_image need it.